apps/docs/pages/guides/getting-started/tutorials/with-vue-3.mdx

```diff
@@ -379,6 +379,119 @@ And then we can add the widget to the Account page:
 </template>
 ```
 
+### Storage management
```

Comment:
Thanks @tvogel!

If you're open to it, you can move this to a "component" in https://github.com/supabase/supabase/tree/master/apps/docs/components/MDX and then you can reuse and reference the content across multiple files like so: https://github.com/supabase/supabase/blob/bdd1f5483d640080c57cb51f210bc6dcef6d4a85/apps/docs/pages/guides/getting-started/tutorials/with-vue-3.mdx?plain=1#L10

Reply:
@dannykng Just done! Fine like that?

Reply:
Looks great @tvogel. Thanks for contributing!